sdk/monitor/monitor-opentelemetry/src/logs/batchLogRecordProcessor.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TraceFlags } from "@opentelemetry/api";
+import { LogRecord, BatchLogRecordProcessor, LogRecordExporter } from "@opentelemetry/sdk-logs";
+
+/**
+ * Azure Monitor BatchLogRecord Processor.
+ * @internal
+ */
+export class AzureBatchLogRecordProcessor extends BatchLogRecordProcessor {
```

Comment:
Just to confirm, this class will never be used directly by user right?

Reply:
It should not be used directly by users, that is correct